src/main/java/com/conveyal/taui/controllers/AggregationAreaController.java

```diff
@@ -56,7 +60,16 @@
             .build();
     private static final FileItemFactory fileItemFactory = new DiskFileItemFactory();
 
-    public static AggregationArea createAggregationArea (Request req, Response res) throws Exception {
+    /**
+     * Create binary .grid files for aggregation (aka mask) areas, save them to S3, and persist their details.
+     * @param req Must include a shapefile on which the aggregation area(s) will be based. If request includes
+     *            "union=true", features will be merged to a single aggregation area with the provided name. If not,
+     *            each feature will be its own aggregation area, named with its value of the attribute specified by
+     *            the provided name.
```

Comment:
How is the name provided? How is the name of that attribute determined? I'm sure we can figure it out by looking at the code but we should state it in the Javadoc. It seems that in the case where `union==true` the HTTP query parameter `name` gives the name of the aggregation area itself, and in the case where `union==false` that same query parameter gives the name of the property containing the name of each feature. This distinction should be explained in the Javadoc. However I think it would be simpler to just not have two different modes.